content/en/docs/setup/independent/high-availability.md

```diff
@@ -316,9 +309,9 @@ the creation of additional nodes could fail due to a lack of required SANs.
         - `ETCD_1_IP`
         - `ETCD_2_IP`
 
-1.  Run `kubeadm init --config kubeadm-config.yaml` on this node.
+1.  Run `kubeadm init --config kubeadm-config.yaml --experimental-upload-certs` on this node.
```

Comment:
@yagonobre I don't remember if certificates for the external etcd are copied automatically or not...
If not, we should add a note that the user must take care of distributing those certs on all control-plane nodes

Reply:
it's upload, we only skip certs upload in case of external ca.